.github/workflows/semgrep.yml

```diff
@@ -26,4 +26,4 @@ jobs:
     if: (github.actor != 'dependabot[bot]')
     steps:
       - uses: actions/checkout@v3
-      - run: semgrep ci
+      - run: semgrep ci --no-suppress-errors
```

Comment:
maybe add a comment about the default failopen and that in a dogfooding context we want to bypass
it to let all errors popup hence the flag below.